sql/moz-fx-data-shared-prod/org_mozilla_broken_site_report/user_reports_live/view.sql

```diff
@@ -23,66 +23,14 @@ WITH live_reports AS (
 SELECT
   document_id AS uuid,
   CAST(submission_timestamp AS DATETIME) AS reported_at,
-  metrics.text2.broken_site_report_description AS comments,
-  metrics.url2.broken_site_report_url AS url,
+  metrics.text.broken_site_report_description AS comments,
+  metrics.url.broken_site_report_url AS url,
```

Comment:
Stage deploys are failing with 
```
Field name text does not exist in STRUCT
```

Only seems to have `text2`

Reply:
Ah, true, the `moz-fx-data-shared-prod.firefox_desktop_live.broken_site_report_v1` table doesn't get the rename-logic that `moz-fx-data-shared-prod.firefox_desktop.broken_site_report` has. urgh. I'll revert the change to the live-table only :(

Reply:
Updated the PR.